rpc/documentation/api.md

```diff
@@ -1146,6 +1147,76 @@ transaction paying to already known addresses or scripts.
 
 ___
 
+#### `UnspentOutputs`
+
+The `UnspentOutputs` method queries the wallet for unspent transaction outputs
+controlled by some account.  Results may be refined by setting a target output
+amount and limiting the required confirmations.  The selection algorithm is
+unspecified.
+
+Output results are always created even if a minimum target output amount could
+not be reached.  This allows this method to behave similar to the `Balance`
+method while also including the outputs that make up that balance.
+
+**Request:** `UnspentOutputsRequest`
+
+- `uint32 account`: Account number containing the keys controlling the output
+  set to query.
+
+- `int64 target_amount`: If positive, the service may limit output results to
+  those that sum to at least this amount (counted in Satoshis).  This may not be
+  negative.
+
+- `int32 required_confirmations`: The minimum number of block confirmations
+  needed to consider including an output in the return set.  This may not be
+  negative.
+
+- `bool include_immature_coinbases`: If true, immature coinbase outputs will
+  also be included.
+
+**Response:** `stream of UnspentOutputResponse`
+
+- `PreviousOutput output`: The unspent output.
+
+  **Nested message:** `PreviousOutput`
+
+  - `bytes transaction_hash`: The hash of the transaction this output originates
+    from.
+
+  - `uint32 output_index`: The output index of the transaction this output
+    originates from.
+
+  - `int64 amount`: The output value (counted in Satoshis) of the unspent
+    transaction output.
+
+  - `bytes pk_script`: The output script of the unspent transaction output.
+
+  - `int64 receive_time`: The earliest Unix time the wallet became aware of the
+    transaction containing this output.
+
+  - `bool from_coinbase`: Whether the output is a coinbase output.
+
+  - `int32 tree`: The tree the output belongs to.  This can take on the values
+    `-1` (invalid), `0` (regular), and `1` (stake).
+
+- `int64 sum`: The sum of all streamed output amounts.  This may be
+  less than a positive target amount if there were not enough eligible outputs
```

Comment:
I would clarify that this is referring specifically to the last message in the stream.